pkg/document/model.go

```diff
@@ -92,6 +143,39 @@ func getUnsortedValueRows(document *yaml.Node, descriptions map[string]helm.Char
 	return createValueRowsFromField("", nil, document.Content[0], descriptions, true)
 }
 
+func getUnsortedSectionedValueRows(valueRows []valueRow) []section {
+	var valueRowsSectionSorted []section
```

Comment:
This variable name makes it sound like they're sorted :wink:

Reply:
True, naming things is hard :D